ecs-cli/modules/cli/compose/entity/entity_helper.go

```diff
@@ -144,18 +144,28 @@ func createRegisterTaskDefinitionRequest(taskDefinition *ecs.TaskDefinition) *ec
 
 // Info returns a formatted list of containers (running and stopped) in the current cluster
 // filtered by this project if filterLocal is set to true
-func Info(entity ProjectEntity, filterLocal bool) (project.InfoSet, error) {
-	containers, err := collectContainers(entity, filterLocal)
+func Info(entity ProjectEntity, filterLocal bool, desiredStatus string) (project.InfoSet, error) {
```

Comment:
Can you also add a test here for the error case on bad status being passed in?

Reply:
Looks like `TestTaskInfoFilterLocal` is failing in the tests...

Reply:
Sure, I added a test in `entity_helper_test.go`